rotkehlchen/rotkehlchen.py

```diff
@@ -429,16 +426,10 @@ def unlock_user(
             btc_derivation_gap_limit=settings.btc_derivation_gap_limit,
         )
 
-        ethereum_accountant = EthereumAccountingAggregator(node_inquirer=ethereum_inquirer, msg_aggregator=self.msg_aggregator)  # noqa: E501
-        optimism_accountant = OptimismAccountingAggregator(node_inquirer=optimism_inquirer, msg_aggregator=self.msg_aggregator)  # noqa: E501
```

Comment:
here we were manually initialing ethereum & optimism AccountingAggregators and were forgetting the other EVM chains.